### DISCUSSION TITLE
Give Multiple Users Ability to Subclaim in a Plot

### DISCUSSION THREAD

**Discussion by ryanchba** [GriefPrevention/GriefPrevention#1215] [Ideas]
Is there a way (besides making the plot an admin plot and making the players admins) so that multiple players are able to create sub-plots within a plot? We've tried /permissiontrust, but with our experiences it's not allowing them to create subdivisions since they dont own the land. Am I just doing it wrong somehow, and /permissiontrust is supposed to make this possible? Thanks!

**Comment by RoboMWM** [comment]
No guarantees if my subclaim addon will do this around v18/v19 timeframe, but currently only the owner can define regions.
